src/style.css

```diff
@@ -1,3 +1,85 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  --header-height: 60px;
+  --blue: #00acdc;
+  background-color: rgb(220, 218, 218);
+  --black: var(--black);
+}
+
+.page__body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
better init box-sizing for all elements in `*` selector

Reply:
Not fixed